setup.py

```diff
@@ -1,10 +1,54 @@
-from setuptools import setup
-
-setup(name="donkey_gym",
-      version="0.1",
-      url="https://github.com/tawnkramer/sdsandbox/src/donkey_gym",
-      author="Tawn Kramer",
-      license="MIT",
-      packages=["donkey_gym"],
-      install_requires = ["gym", "numpy", 'pillow']
-      )
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+from gym_donkeycar import __version__
+
+"""The setup script."""
+
+description = 'OpenAI Gym Environments for Donkey Car'
+
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read()
+
+requirements = [
+    'gym',
+    'numpy',
+    'pillow'
+]
+
+setup_requirements = ['pytest-runner', ]
+
+test_requirements = ['pytest', ]
+
+setup(
+    author='Tawn Kramer',
+    author_email='leigh@data-literate.com',
```

Comment:
@tawnkramer `author_email` is required for PyPi upload, so I left mine here. Let me know if you have an email you'd like to use. Note: this is public, so it should not be an email you use as a login/username.

Reply:
tawnkramer@gmail.com is my email. tawnkramer is the user account. I have two-factor auth enabled here, so think we are ok.